jack-core/src/com/rapleaf/jack/DatabaseConnection.java

```diff
@@ -101,21 +101,39 @@ public DatabaseConnection(String dbname_key, long expiration, String driverClass
    * Get a Connection to a database. If there is no connection, create a new one.
    * If the connection hasn't been used in a long time, close it and create a new one.
    * We do this because MySQL has an 8 hour idle connection timeout.
+   *
+   * Because of the potential downtime of CloudSQL, it implements exponential retry policy.
```

Comment:
nit: perhaps a more general description about handling intermittent MySQL downtime would be more broadly applicable than a statement regarding the specific database service that LiveRamp uses (CloudSQL)?